testbeds/AL-240-156.robot

```diff
@@ -0,0 +1,134 @@
+
```

Comment:
Remove this file we don't want to add it to GitHub

Instead just change Testbed.robot file to have the new line(s)

Reply:
All changes done by rebase, edit, add, commit, fetch.